Introduction/Introduction.md

```diff
@@ -0,0 +1,23 @@
+Kernel introduction.
+================================================================================
+
+The Begining
+--------------------------------------------------------------------------------
+Linux has began in the Operating system more important in the last years and many people use it in their lives commonly.
+But nobody like remember the history about the Linux kernel.
+
+It s now installed on millions of systems and used by home users and professionals alike for a wide range of tasks. From miniature embedded systems in wristwatches to massively parallel mainframes, the are countless way of exploiting linux productively.
+
+This the first part to undestand if you want to begin with this topics.
+Linux looks like any other Unix systems; indeed, UNIX compatiblity has a major design and goal of the Linux Kernel project and it is considered as the core of the Operating Systems.
+The First difference betwen any other programs in the system consist that controls the hardware and the applications but the applications do not communicate with the hardware directly, instead the go to the kernel. In summary
```

Comment:
- **f**irst (not capitalized)
- betw**e**en
- The sentences is hard to understand. What exactly do you mean here?